UNRELEASED_CHANGELOG.md

```diff
@@ -33,6 +33,7 @@
 
 ### ⚠️ Changed
 - Deprecated `Channel#name`, `Channel#image`, `User#name`, `Ues#image` extension properties. Use class members instead.
+- 🚨 Breaking change: Now it is possible to hard delete messages. Insert a flag `hard = true` in the `ChatClient.deleteMessage` and it will be deleted in the backend. **This action can't be undone!**
```

Comment:
Is this a breaking change? 🤔

Reply:
I'll add the `@JvmOverloads` suggested by Dmitri and it won't be a breaking change. (It would be a breaking change for Java users)